stack.yaml

```diff
@@ -23,7 +23,7 @@ extra-package-dbs: []
 
 # Control whether we use the GHC we find on the path
 nix:
-  enable: true
+  enable: false
```

Comment:
I suspect this is a controversial change. I am not familiar enough with nix to configure GitHub actions to build using nix. I always disable this section when building ffmpeg-light on my local machine.

Reply:
No worries! Let's get one path through CI sorted out, and a nix build can be sorted out later.